dataset_loaders/parallel_loader.py

```diff
@@ -808,7 +824,17 @@ def __config_parser__(self):
                                'path.' % config_path)
         config_parser.read(config_path)
         return config_parser
-
+    
+    @classproperty
+    def datasets_local_path(self):
+        config_parser = self.__config_parser__
+        return config_parser.get('general', 'datasets_local_path')
+    
+    @classproperty
+    def is_open_permissions(self):
+        config_parser = self.__config_parser__
+        return config_parser.get('general', 'open_permissions')
```

Comment:
Please return a boolean with `config_parser.get('general', 'open_permissions') in ['True', 'true']` and update the property name according to the config flag name you choose.